src/hooks/use-location-sync.ts

```diff
@@ -72,9 +72,9 @@ export default function useLocationSync({
   ]);
 
   const syncLocation = useCallback(
-    (defaultState: DeckView) => {
+    (defaultState: DeckView): DeckView => {
       if (disableInteractivity) {
-        return null as unknown as DeckView;
+        return defaultState;
```

Comment:
Before TS conversion ,this was just `return;`.  I mistakingly thought changing to `null` would be inconsequential.  
But returning `defaultState` is a better option.

Reply:
I didn't trace through all the code paths, but I did have this thought of "shouldn't TS be catching this for us?" 😛

Reply:
Yeah, but little did you know I was doing the classic Jedi-mind-trick on TypeScript! 
"null is a DeckView"